efibuild.sh

```diff
@@ -17,7 +17,14 @@ prompt() {
 
 updaterepo() {
   if [ ! -d "$2" ]; then
-    git clone --recursive "$1" -b "$3" --depth=1 "$2" || exit 1
+      if [ "$2" == "UDK" ] ; then
```

Comment:
Why do we make UDK any special? This can be done for any repository, no?

Reply:
It can actually ... only did like that to minimise scope of impact of changes when things were up in the air.

Reply:
Please change that then, and also make sure to maintain the same indentation as before.

Reply:
Manually copied existing indentations but better if you add `.editorconfig` files to the repos so that your style can be picked up by IDEs.

PS. moved the update line so that it applies to new cloning and updates.